src/content/docs/errors-inbox/errors-inbox.mdx

```diff
@@ -106,14 +106,33 @@ The New Relic managed rules normalize the error data, identifying and ignoring u
     id="similar-events"
     title="Troubleshooting: similar looking events do not group together"
   >
-    If you see "like" error events grouped into different error groups incorrectly, try removing the unique identifier from the error class or message and store those as attributes instead. This will allow you to more easily facet on the attribute values and reduce the number of error groups.
+    If you see "like" error events grouped into different error groups incorrectly, review our best practices section for guidance on how to adjust error messages. <!-- Hyperlink to the best practices section please --> 
```

Comment:
@homelessbirds I think the <!--- --> stuff is the culprit

Reply:
Add the link they're asking for and then delete `<!-- Hyperlink to the best practices section please -->`